pkg/api/validation/validation.go

```diff
@@ -606,8 +606,8 @@ func validateCinderVolumeSource(cd *api.CinderVolumeSource) validation.ErrorList
 	if cd.VolumeID == "" {
 		allErrs = append(allErrs, validation.NewFieldRequired("volumeID"))
 	}
-	if cd.FSType == "" || (cd.FSType != "ext3" && cd.FSType != "ext4") {
-		allErrs = append(allErrs, validation.NewFieldRequired("fsType required and should be of type ext3 or ext4"))
+	if cd.FSType == "" {
+		allErrs = append(allErrs, validation.NewFieldRequired("fsType"))
```

Comment:
I just noticed this. fsType is not required. mount.SafeFormatAndMount will default to ext4. In fact the parameter has "omitempty" in `types.go` even though the comment says required.